server/auth/auth_test.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="access-token" content="{{.AccessToken}}" />
+    <meta name="id-token" content="{{.IDToken}}" />
+    <meta name="user-name" content="{{.UserName}}" />
+    <meta name="user-email" content="{{.UserEmail}}" />
+    <meta name="user-picture" content="{{.UserPicture}}" />
+    <meta name="return-url" content="{{.ReturnUrl}}" />
+    <meta name="public-path" content="{{.PublicPath}}" />
+  </head>
+</html>
```

Comment:
Just thinking out loud headers may be a good way to send this data back in the callback instead of the meta of the html document.

Reply:
i want to revisit this particular piece (sending access-token to client) because when returning an html we are limitied to only the current domain and can't serve as a true API only server when it comes to auth endpoint

Particularly 302 redirect users to the final returnUrl and set a cookie with users data. Will have to break down the access-token into 4kb cookie items to not exceed the size limit

Reply:
done ^